tests/integration/dhcp_test.py

```diff
@@ -683,6 +678,17 @@ def test_slave_ipaddr_learned_via_dhcp_added_as_static_to_linux_bridge(
     assertlib.assert_state(bridge_desired_state)
 
 
+@pytest.mark.xfail(raises=NmstateNotImplementedError)
```

Comment:
I think re-ordering the test is unrelated to the other issue fixed here, and that it deserves a separate commit.

Not that picky though.

Reply:
It is related because without it all tests below it fail. Relocating it in its own commit is possible, but then it will have no context/reason.

Reply:
Fair enough, as I said, won't block this because of that.